src/components/List.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class List extends React.PureComponent {
+  render() {
+    const { children, ...props } = this.props;
+
+    return (
+      // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
ou did a great job! However, there are some places that we need to work on 😉

Students are not allowed to use such techniques. Instead of using <ul {...props}>, you can replace assign props to const value example of correct usage const {src, alt} = props;